chronicles-rulebook.md

```diff
@@ -1842,13 +1841,6 @@ Characters may alternatively choose to explicitly memorize their spell list. Doi
 | **Incant**     | "Fiends of the dark nether, create a blood sacrifice." |
 | **Description** |  This spell prevents the target from being healed. This spell only affects the ability to be healed. It does not affect spells that purify or restore. If a spell does both healing and restoration effects, only the healing part of the spell fails. |
 
-##### **Close Wounds**
-| **Spell**            | **Close Wounds** |
```

Comment:
Are we removing BLEED in a future update to go along with this removal?

Reply:
Already done. This was a change that was made previously and there were just some lingering things that got missed. BLEED should be removed from status, which I'll do